armi/reactor/tests/test_composites.py

```diff
@@ -473,6 +473,28 @@ def test_getReactionRates(self):
             self.assertAlmostEqual(rRatesCore[key], val)
             self.assertAlmostEqual(rRatesReactor[key], val)
 
+    def test_getFirstComponent(self):
+        c = self.container.getComponents()[0]
+        c0 = self.container.getFirstComponent()
+        self.assertEqual(c, c0)
```

Comment:
Stronger check that we get the same objects, not identical objects
```suggestion
        self.assertIs(c, c0)
```